docs/install.md

```diff
@@ -263,6 +263,7 @@ $ kubectl create -f https://github.com/intel/pmem-csi/raw/devel/deploy/crd/pmem-
 ``` console
 $ kubectl create -f https://github.com/intel/pmem-csi/raw/devel/deploy/operator/pmem-csi-operator.yaml
 ```
+The operator gets deployed in 'pmem-csi' namespace.
```

Comment:
"in a namespace called 'pmem-csi'" which gets created by that YAML file"

Reply:
Changed the text as suggested.